src/client/sandbox/native-methods.ts

```diff
@@ -520,7 +520,7 @@ class NativeMethods {
         this.hasAttributeNS                = nativeElement.hasAttributeNS;
         this.hasAttributes                 = nativeElement.hasAttributes;
         this.anchorToString                = win.HTMLAnchorElement.prototype.toString;
-        this.matches                       = nativeElement.matches || nativeElement.msMatchesSelector;
+        this.matches                       = nativeElement.matches || (nativeElement as HTMLElement as IEHTMLElement).msMatchesSelector;
```

Comment:
It's unnecessary type casting, see comments for the `src/typings/extend-html-element.d.ts` file below.